### PR TITLE
Fix translations

### PR DESCRIPTION
**Pull request by xuebingli**
@zendesk/vegemite @zendesk/i18n @pmgrove @fbvilela 

This PR fixes translations in an invalid format or with duplicates. Specifically

- For `en.json` and `en.yml`, remove duplicate translation `parameters.description` in favour of `short_description`
- For all other locales, promote `parameters.description` to top-level marketplace translation `description`
- Add `.zat`

Risks
- [low] description not being displayed or displayed in a wrong locale

### CODE REVIEW

File: .zat [hunk 1]
```diff
@@ -0,0 +1,4 @@
+{
+  "zat_latest": "2.3.0",
+  "zat_update_check": "2017-09-22"
+}
```
Comment (mtsolakiszen) [inline .zat]: New line

File: translations/cs.json [hunk 2]
```diff
@@ -1,15 +1,15 @@
 {
   "app": {
     "parameters": {
+      "description": "Jakýkoli text, který chcete, se zástupnými texty.",
```
Comment (seangoedecke) [inline translations/cs.json]: Should this be top-level?

File: translations/en-US.json [hunk 3]
```diff
@@ -1,15 +1,15 @@
 {
   "app": {
     "parameters": {
+      "description": "Any text you want, with data placeholders.",
```
Comment (seangoedecke) [inline translations/en-US.json]: Should this be top-level?

File: .zat (hunk 1 above)
Comment (xuebingli) [reply .zat]: This file is automatically generated. If a new line is added, it will be removed on the next run of `zat validate`.

**Comment by xuebingli** [comment]
@mtsolakiszen 

> Do we have fixtures to test what gets priority `description` vs `short_description`?

`short_description` get priority and, in case `short_description` does not exist, `description` will be used as fallback.

**Pull request merged**